sympy/utilities/misc.py

```diff
@@ -355,7 +355,8 @@ def translate(s, a, b=None, c=None):
 
     mr = {}
     if a is None:
-        assert c is None
+        if c is not None:
+            raise TypeError('Called translate with 4 arguments and a second argument of None, expected 3 arguments if second argument is None')
```

Comment:
OK. In my project I also create longer line, but in SymPy I've never seen line longer than 80. For example, look at lines `20:43` in this file, `misc.py`.